tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -592,10 +563,12 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
         client.Config.Namespace = currentNamespace
 
     } else if artifact != "" {
-        stat, err := os.Stat(artifact)
+        ext := filepath.Ext(artifact)
+
+        _, err := os.Stat(artifact)
         if err != nil {
             whisk.Debug(whisk.DbgError, "os.Stat(%s) error: %s\n", artifact, err)
-            errMsg := fmt.Sprintf("File '%s' does not exist: %s", artifact, err)
+            errMsg := fmt.Sprintf("File '%s' does not exist.", artifact)
```

Comment:
os.Stat() errors can mean either 1) the path does not exist or 2) some other error.
http://stackoverflow.com/questions/10510691/how-to-check-whether-a-file-or-directory-denoted-by-a-path-exists-in-golang